src/llv8.cc

```diff
@@ -457,6 +458,11 @@ std::string HeapObject::Inspect(bool detailed, Error& err) {
     return pre + fn.Inspect(detailed, err);
   }
 
+  if (type == v8()->types()->kJSRegExpType) {
+    JSRegExp re(this);
+    return pre + re.Inspect(detailed, err);
```

Comment:
That is my plan. I am still trying to wrap my head around all of it. The current inspection looks like:

```
(lldb) v8 i 0x00002c0eb9123c69
0x00002c0eb9123c69:<Object: RegExp properties {
    .source=0x0000105f9260a241:<String: "blahblah">,
    .global=0x00001253415043c9:<false>,
    .ignoreCase=0x00001253415043c9:<false>,
    .multiline=0x00001253415043c9:<false>,
    .lastIndex=<Smi: 0>}>
```

It just walks the properties of the object. Before, it just showed unavailable.